pkg/util/gce/gce_tags_test.go

```diff
@@ -40,10 +50,44 @@ func mockMetadataRequest(t *testing.T) *httptest.Server {
 func TestGetHostTags(t *testing.T) {
 	server := mockMetadataRequest(t)
 	defer server.Close()
+	defer cache.Cache.Delete(tagsCacheKey)
 	tags, err := GetTags()
-	if err != nil {
-		assert.Fail(t, fmt.Sprintf("Error getting tags: %v", err))
+	require.Nil(t, err)
+
+	expectedTags := []string{
+		"tag",
+		"zone:us-east1-b",
+		"instance-type:n1-standard-1",
+		"internal-hostname:dd-test.c.datadog-dd-test.internal",
+		"instance-id:1111111111111111111",
+		"project:test-project",
+		"numeric_project_id:111111111111",
+		"cluster-location:us-east1-b",
+		"cluster-name:test-cluster",
+		"created-by:projects/111111111111/zones/us-east1-b/instanceGroupManagers/gke-test-cluster-default-pool-0012834b-grp",
+		"gci-ensure-gke-docker:true",
+		"gci-update-strategy:update_disabled",
+		"google-compute-enable-pcid:true",
+		"instance-template:projects/111111111111/global/instanceTemplates/gke-test-cluster-default-pool-0012834b",
 	}
+	require.Len(t, tags, len(expectedTags))
+	for _, tag := range tags {
+		assert.Contains(t, expectedTags, tag)
+	}
```

Comment:
I think it would be nice if you can factorize the logic in a function or factorize`expectedTags`.